sdk/how-to-guides/getting-started-with-testing.md

```diff
@@ -17,97 +16,85 @@ What you will learn:
   - Where can you apply functional testing?
 - How to automate this in a CI pipeline.
 
+# Charmcraft Profiles
 
-# Templates
-
-The two most popular ways to set up a charm project are via `charmcraft init` and by using the [`template-operator`](https://github.com/canonical/template-operator) template repo. 
-[note]
-Currently, so-called charmcraft profiles are being developed -- these will consolidate some standard templates and make them available via `charmcraft init --profile <profile name>`.
-[/note]
+The most popular way to set up a charm project is via `charmcraft init`. For example, to get set up for creating a machine charm, run:
 
-## Charmcraft
-The template you get with `charmcraft init` is rather compact. The testing infrastructure consists of a `tests/test_charm.py` file, and a `run_tests` shell script that contains:
-```bash
-                                                      
-if [ -z "$VIRTUAL_ENV" -a -d venv/ ]; then            
-    . venv/bin/activate                               
-fi                                                    
-                                                      
-if [ -z "$PYTHONPATH" ]; then                         
-    export PYTHONPATH="lib:src"                       
-else                                                  
-    export PYTHONPATH="lib:src:$PYTHONPATH"           
-fi                                                    
-                                                      
-flake8                                                
-coverage run --branch --source=src -m unittest -v "$@"
-coverage report -m                                    
+```shell
+charmcraft init --profile=machine
```

Comment:
This feels out of scope for the doc.

Reply:
I was trying to do a minimal change (before the rethinking/refactoring project), and so just swapped out the previous (outdated) template content for charmcraft content.